### PR TITLE
Milestone 01:Homepage Setup

### PR DESCRIPTION
**Pull request by borrnelsonk**
In this milestone we made following changes:

1. Added the markup for the default homepage.
2. Added the basic styling to the homepage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <p>Hello Microverse</p>
```
Comment (prolajumokeoni) [inline index.html]: - [x] Please use a h1 tag based on the [requirements ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements)

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @borrnelsonk, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR title and Description



## Required Changes ♻️

- [x] Kindly add a test.md file with the text "This file should be ignored by git", this file should be listed in `.gitignore` so it is ignored by git 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (borrnelsonk) [reply index.html]: Thankyou @prolajumokeoni for your review. I have made the proposed changes as summarized below;

1. Added test.md and set it to be ignored by git.
2. Used h1 tag as per the requirements.

**Review by prolajumokeoni — APPROVED** [review]
Hi @borrnelsonk 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**